docs/gallery/plot_external_resources.py

```diff
@@ -320,3 +323,34 @@ def __init__(self, **kwargs):
 
 er_read = ExternalResources.from_norm_tsv(path='./')
 os.remove('./er.zip')
+
+###############################################################################
+# Using TermSet with ExternalResources
+# ------------------------------------------------
+# :py:class:`~hdmf.TermSet` allows for an easier way to add references to
```

Comment:
```suggestion
# :py:class:`~hdmf.term_set.TermSet` allows for an easier way to add references to
```
The link in the generated docs for this PR does not work using :py:class:`~hdmf.TermSet`. I think you need to add the name of the module.